pkg/platform/packet/packet_test.go

```diff
@@ -150,103 +150,70 @@ func TestValidateOSVersion(t *testing.T) {
 	}
 }
 
-func TestCheckResFormatPrefixValidInput(t *testing.T) {
-	r := map[string]string{"worker-2": "", "worker-3": ""}
-
-	if d := checkResFormat(r, "", "", "worker"); d.HasErrors() {
-		t.Errorf("Validation failed and shouldn't for: %v", r)
-	}
-}
-
-func TestCheckResFormatPrefixInvalidInput(t *testing.T) {
-	r := map[string]string{"broken-1": "", "worker-1-2": "", "worker-a": ""}
-
-	if d := checkResFormat(r, "", "", "worker"); !d.HasErrors() {
-		t.Errorf("Should fail with res: %v", r)
-	}
-}
-
-func TestCheckEachReservationValidInput(t *testing.T) {
-	cases := []struct {
-		role           nodeRole
-		resDefault     string
-		reservationIDs map[string]string
+//nolint:funlen
+func TestCheckValidConfig(t *testing.T) {
+	cases := map[string]struct {
+		mutateF     func(*config)
+		expectError bool
 	}{
-		{
-			// Test validates worker config.
-			role: worker,
-			reservationIDs: map[string]string{
-				"worker-2": "",
-				"worker-3": "",
-			},
+		"base_config_is_valid": {
```

Comment:
So you can find test case by message printed by `go test`:
```
--- PASS: TestCheckValidConfig (0.00s)
    --- PASS: TestCheckValidConfig/not_sequential_reservation_IDs_are_invalid (0.00s)
    --- PASS: TestCheckValidConfig/reservation_IDs_for_worker_nodes_must_be_prefixed_with_'worker' (0.00s)
    --- PASS: TestCheckValidConfig/reservation_IDs_must_be_sequential (0.00s)
    --- PASS: TestCheckValidConfig/reservation_IDs_for_controller_nodes_must_be_prefixed_with_'controller' (0.00s)
    --- PASS: TestCheckValidConfig/reservation_IDs_for_worker_nodes_can't_have_random_prefix (0.00s)
    --- PASS: TestCheckValidConfig/reservation_IDs_for_worker_nodes_can't_be_mixed_default_reservation_ID (0.00s)
    --- PASS: TestCheckValidConfig/reservation_IDs_for_worker_nodes_can't_be_set_to_'next-available' (0.00s)
    --- PASS: TestCheckValidConfig/reservation_IDs_can't_be_empty (0.00s)
    --- PASS: TestCheckValidConfig/base_config_is_valid (0.00s)
    --- PASS: TestCheckValidConfig/reservation_IDs_for_controller_nodes_can't_have_random_prefix (0.00s)
```

Otherwise you have `base config is valid` in code and `base_config_is_valid` is printed. Not very convenient.

Reply:
ah, so `go test` adds the underscores? Fair then.